frontend/packages/integration-tests-cypress/tests/i18n/pseudotranslation.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { checkErrors } from '../../support';
+import { nav } from '../../views/nav';
+import { timestamp } from '../../views/timestamp';
+
+describe('Localization', () => {
+  before(() => {
+    cy.login();
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  it('pseudolocalizes navigation', () => {
+    cy.log('test navigation');
+    cy.visit('/dashboards?pseudolocalization=true&lng=en');
+    nav.sidenav.shouldMatchRegex(/\[[αḅͼḍḛϝḡḥḭĵḳḽṃṇṓṗʠṛṡṭṵṽẁẋẏẓḀḂḈḌḚḞḠḤḬĴḰḺṀṄṎṔǪṚṢṪṲṾŴẊŶŻ]+\]/);
```

Comment:
```suggestion
    nav.sidenav.shouldMatchRegex(/\[[^a-zA-Z]+\]/);
```

We should make this a reusable helper.

Reply:
I tried the [^a-zA-Z] approach yesterday in one of those regex testing websites and it wasn't passing.